docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -236,24 +231,27 @@ Component names in the theme have changed as well:
 
 -MuiYearPicker: {
 +MuiYearCalendar: {
+
+-MuiClockPicker: {
++MuiTimeClock: {
 ```
 
 ### Rename `date` prop to `value` on view components
 
-The `date` prop has been renamed `value` on `MonthPicker`, `YearPicker`, `ClockPicker` and `CalendarPicker`:
+The `date` prop has been renamed `value` on `MonthCalendar`, `YearCalendar`, `TimeClock` and `DateCalendar`:
```

Comment:
I'm using the new component names here, I take for granted that people read the section above
What do you think ?